native/Makefile

```diff
@@ -373,19 +383,26 @@ endif
 
 
 ifeq ($(OS),darwin)
-XCODE_VERSION=$(shell xcodebuild -version | grep Xcode | sed 's/^Xcode \([1-9]\).*/\1/g')
+XCODE_VERSION=$(shell xcodebuild -version | grep Xcode | sed 's/^Xcode \([1-9][0-9]*\).*/\1/g')
```

Comment:
Xcode became two digit?

Reply:
Yes, the newest version of Xcode is 12:
https://developer.apple.com/documentation/xcode-release-notes/xcode-12-beta-release-notes